src/Presence/apple.json

```
@@ -0,0 +1,282 @@
+    },
+    "apple.support": {
+        "message": "Apple - Support",
+        "description": ""
+    },
+    "apple.supportTwo": {
+        "message": "Apple Support",
+        "description": ""
+    },
+    "apple.supportArticle": {
+        "message": "Apple Support - Article:",
+        "description": "Support article"
+    },
+    "apple.appStoreCon": {
+        "message": "Connecting...",
+        "description": "Connecting to the App Store"
+    },
+    "apple.iCloudMail": {
+        "message": "Mail",
+        "description": "iCloud mail service"
+    },
+    "apple.iCloudContacts": {
+        "message": "Contacts",
+        "description": "iCloud contacts"
+    },
+    "apple.iCloudCalendar": {
+        "message": "Calendar",
+        "description": "iCloud calendar"
+    },
+    "apple.iCloudPhotos": {
+        "message": "Photos",
+        "description": "iCloud photos"
+    },
+    "apple.iCloudNotes": {
+        "message": "Notes",
+        "description": "iCloud notes"
+    },
+    "apple.iCloudReminders": {
+        "message": "Reminders",
+        "description": "iCloud reminders"
+    },
+    "apple.iCloudPagesCreate": {
+        "message": "Creating...",
+        "description": "Creating a new Pages document"
+    },
+    "apple.iCloudKeynoteWait": {
+        "message": "Waiting room...",
+        "description": ""
+    },
+    "apple.devNew": {
+        "message": "What's new",
+        "description": "New things"
+    },
+    "apple.devSubmit": {
+        "message": "Submissions",
+        "description": "Submit iOS and iPadOS apps to the App Store"
+    },
+    "apple.devPhonePad": {
+        "message": "iPhone and iPad apps",
+        "description": ""
+    },
+    "apple.devDistribution": {
+        "message": "Distributing software",
+        "description": ""
+    },
+    "apple.devDocs": {
+        "message": "Documentation",
+        "description": "Developer documentation"
+    },
+    "apple.devHIG": {
+        "message": "Human Interface Guidelines",
+        "description": ""
+    },
+    "apple.devResources": {
+        "message": "Resources",
+        "description": "Apple Design Resources"
+    },
+    "apple.devFeatures": {
+        "message": "Features",
+        "description": ""
+    },
+    "apple.devGames": {
+        "message": "Games",
+        "description": "Games like Minecraft etc."
+    },
+    "apple.forumThread": {
+        "message": "Forum - Thread",
+        "description": ""
+    },
+    "apple.forumTags": {
+        "message": "Forum - Tags",
+        "description": ""
+    },
+    "apple.forumProfile": {
+        "message": "Forum - Profile",
+        "description": "User profile"
+    },
+    "apple.forumPreferences": {
+        "message": "Preferences",
+        "description": ""
+    },
+    "apple.forumCreateThread": {
+        "message": "Creating thread",
+        "description": ""
+    },
+    "apple.forumRegister": {
+        "message": "Registration",
```

Why make it complicated (I assume there is another string showing up at the same time as this one saying this relates to the Forums)
```suggestion
        "message": "Registering",
```

---

Why is **Registration** not okay?

---

Because it's unnecessarily complicated and hard to translate in most languages

---

I've changed the description